ElfPatcher.cs

```diff
@@ -8,8 +8,17 @@
 
 namespace ToVPatcher {
 	class ElfPatcher {
+		public static bool linuxOrMac = false;
 		public static void PatchElf( string ebootPath, string patchDir, string outDir, string outMd5 = null, BackgroundWorker worker = null ) {
-			string ebootModPath = Path.GetFullPath( "ebootmod/ebootMOD.exe" );
+			OperatingSystem os = Environment.OSVersion;
+			PlatformID pid = os.Platform;
+			string patcherPath = "";
+			if (pid == PlatformID.Unix || pid == PlatformID.MacOSX) {
```

Comment:
It's probably a better idea to check this the other way around. Most OSes should be able to use unself, it's just Windows we special-case.